firebase.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
does this need to be at the root of the repo? We should try not to pollute the root if we can help it

Reply:
Moved to `site` for now.